cluster/container.go

```diff
@@ -54,6 +54,19 @@ func FullStateString(state *types.ContainerState) string {
 		if state.Restarting {
 			return fmt.Sprintf("Restarting (%d) %s ago", state.ExitCode, units.HumanDuration(time.Now().UTC().Sub(finishedAt)))
 		}
+		// Container is Up. Add Health check info when healthcheck is defined.
+		healthText := ""
+		if h := state.Health; h != nil {
+			switch h.Status {
+			case types.Starting:
+				healthText = "health: starting"
+			default: // Healthy and Unhealthy are clear on their own
+				healthText = h.Status
+			}
+		}
+		if len(healthText) > 0 {
```

Comment:
In that case container is up and running. It's just failing health check. So I think it's a reasonable combination.